### ISSUE TITLE
Maven warning for com.sun.xml.bind on JDK11

### ISSUE THREAD

**Issue by MahatmaFatalError** [SAP/cf-java-logging-support#57]
I am using latest version v3.0.7 on SapMachine 11.0.6.0.1+10-LTS.

`mvn clean install `(-X for debug output) leads to the following output:

```
[WARNING] The POM for com.sun.xml.bind:jaxb-core:jar:2.2.11 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
[WARNING] The POM for com.sun.xml.bind:jaxb-impl:jar:2.2.11 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
```
with debug info
```
[WARNING] The POM for com.sun.xml.bind:jaxb-core:jar:2.2.11 is invalid, transitive dependencies (if any) will not be available: 1 problem was encountered while building the effective model for com.sun.xml.bind:jaxb-core:2.2.11
[ERROR] 'dependencyManagement.dependencies.dependency.systemPath' for com.sun:tools:jar must specify an absolute path but is ${tools.jar} @

[WARNING] The POM for com.sun.xml.bind:jaxb-impl:jar:2.2.11 is invalid, transitive dependencies (if any) will not be available: 1 problem was encountered while building the effective model for com.sun.xml.bind:jaxb-impl:2.2.11
[ERROR] 'dependencyManagement.dependencies.dependency.systemPath' for com.sun:tools:jar must specify an absolute path but is ${tools.jar} @
```

A similar issue was solved here https://github.com/authzforce/restful-pdp/issues/4#issuecomment-396642962 by replacing it with the post-java8 dependencies.
```
<dependency>
    <groupId>javax.xml.bind</groupId>
    <artifactId>jaxb-api</artifactId>
    <version>2.3.0</version>
</dependency>
<dependency>
    <groupId>org.glassfish.jaxb</groupId>
    <artifactId>jaxb-runtime</artifactId>
    <version>2.3.0</version>
    <scope>runtime</scope>
</dependency>
```

**Comment by KarstenSchnitter** [comment]
Thanks, for this information. This dependency was introduced to allow Java 11 builds and was tested with an OpenJDK version 11.0.2 using Maven 3.5.3. With this setup, I get no warnings. I will update the dependencies to the newer version to eliminate this problem.

**Comment by KarstenSchnitter** [comment]
I updated the dependencies with https://github.com/SAP/cf-java-logging-support/commit/142c70d01cb8421c1f446a4d78afeff284be35af. Can you try your build with this commit? I will create a new release from there, if this solves the issue.

**Comment by MahatmaFatalError** [comment]
>  Can you try your build with this commit? 

How can I do that?

**Comment by KarstenSchnitter** [comment]
There is no SNAPSHOT release yet. You could check out this repository and create one. That would be a great help.

**Comment by MahatmaFatalError** [comment]
Yep, seems to work. At least, now, there is a different dependency listed that still has this old transitive dependency.
BTW, would have been nice to have a property in the parent pom for the version that gets reused in the sub-modules ;)

**Comment by KarstenSchnitter** [comment]
Fixed in v3.0.8
